### PR TITLE
Add concert class

### LINKED ISSUE #45: Create Concert class
Concert is a class that has 

- concert name
- address 
- date and time

Should be similar to Person class with different fields

### PR DESCRIPTION
**Pull request by wes-w-z-h**
Fixes #45

### CODE REVIEW

File: src/main/java/seedu/address/model/concert/UniqueConcertList.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        }
+        internalList.add(toAdd);
+    }
+
+    /**
+     * Replaces the concert {@code target} in the list with {@code editedconcert}. {@code target}
+     * must exist in the list. The concert identity of {@code editedconcert} must not be the same as
+     * another existing concert in the list.
+     */
+    public void setconcert(Concert target, Concert editedconcert) {
```
Comment (bbbbcccd) [inline src/main/java/seedu/address/model/concert/UniqueConcertList.java]: camel case method name

File: src/test/java/seedu/address/model/concert/ConcertTest.java
Comment (bbbbcccd) [inline src/test/java/seedu/address/model/concert/ConcertTest.java:1]: A bit lacking but okay for now. I'll open a new issue for this

**Review by bbbbcccd — COMMENTED** [review]
lgtm with minor changes

**Review by bbbbcccd — APPROVED** [review]
lgtm!

**Comment by wes-w-z-h** [comment]
Fixes #49

**Pull request merged**